### DISCUSSION TITLE
BackupBox - off-grid mode issue (question) [Solved]

### DISCUSSION THREAD

**Discussion by 0k00l** [wlcrs/huawei_solar#674] [General]
Hi, don't know where to look for my problem. So i try to ask you guys. 
In short: 
_I have SUN2000-6KTL-M1 with DTS666-H 250A smart power sensor and backupbox B1 (three-phase) without any energy storage. Backupbox is connected to 2 energy sockets that works when i'm in on-grid mode but when i try to force off-grid (via main circuit breaker or in fussionsolar app) i get alarm: **On-Grid/Off-Grid Controller Error** and inverter doesn't switch to off-grid (sockets has no power). Backupbox is connected exactly as they(huawei) mention in manual.
How can i make those sockets works in off-grid?_ 
Don't know what is going on - tried everything without luck. 
Any help will be appreciated! ;)

**Comment by Roving-Ronin** [comment]
To get off grid mode to work with a BackupBox..... Solution:

Buy a LUNA ESS Battery and attach it to the M1 inverter !

**Reply by 0k00l** [reply]
So battery is mandatory? They didn't mention it in manual :/. 
No other way to enable it (without LUNA)?

**Reply by Roving-Ronin** [reply]
@0k00l  Yes.  There is the theoretical capability to do 'PV Backup'  (run off solar panels) however it is NOT recommended to do, least of which is for the simplest issue that as cloud pass the inverter output will drop, so you can end up running (say) 5kW load, then inverters drop to only 2kW and all your circuits trip and appliances fail. Repeatedly doing this is not a good idea for for electrical appliances.

It is due to this the normal 'steps' in deploying solar are:
1) Install solar inverter + panels, and (should be mandatory) a power meter.
2) If you can afford it, then install batteries.
3) If you have issues with recurring grid outages (or have the cash to splash) then install BackupBox to work during a grid outage.
... somewhere in there also add in an EV charger if needed.

Trying to run your house without a battery, but with a BackupBox + Solar PV, is like wondering why TV isn't working when you pull the power plug from the wall.

**Reply by 0k00l** [reply]
Got it - so i should install energy storage. Is there any cheaper alternatives for LUNA that will work in my case(compatible with SUN2000-6KTL-M1)?

My plan was like this (without storage): if grid fails i would have access to some energy from plug (i have UPS already, 2.5 kWh and i thought that i could plug it into those sockets from backupbox - it will charge during day and works as buffer for some time)

**Reply by Roving-Ronin** [reply]
LUNA is 'THE' battery for Huawei inverters. Prior to Huawei LUNA they did support LG batteries, but that option was superceded with the LUNA ESS launch (and LG has MANY issues and even recently had another recall, and fires).

If you have a 2500W UPS it will NOT support plugging it into the backup box. If your running some PCs or GPO connected devices, you could use the UPS as normal and plug them into the UPS and have them run for a while when the grid is down. 
If you want the inverter + backupbox to work, then buy a LUNA ESS battery and you can protect ONE circuit (at max. 3.3kW).

**Reply by 0k00l** [reply]
Thank You very much for help.

You mention that off-grid is possible without LUNA - could you share a way to do this? I don't know if i will gather required money for LUNA, so it would be my Plan B ;)

**Reply by Roving-Ronin** [reply]
@0k00l

No Huawei LUNA ESS + BackupBox = No Off-Grid during an outage. If you want a backup capability during an outage, then the only other alternative is to hire a qualified electrician to design a solution, that includes a manual circuit breaker to disconnect the main feed from the power grid and allow installation of a generator (preferably 3 phase, given your house is 3p).

**Reply by 0k00l** [reply]
Ok - so there is no hope then LUNA :). 

Thank you very very much for patience and helpful explanations.

**Reply by Albarraque** [reply]
I have a SUN2000-6KTL with 5 Sun 2000 (25 kW) units and a backup box in a single-phase installation. When I use the system in off-grid mode, the system shuts down and restarts in two circumstances:
- At the start of the day (between 8 am and 9 am) with intermittent on/off loads (air fryer) and batteries at 50 per cent
- When the batteries are at 100 per cent with intermittent on/off loads (air fryer, iron, microwave); if I switch on the electric car, it does not switch off
